src/nvim/edit.c

```diff
@@ -7781,7 +7781,7 @@ int hkmap(int c)
       if (c < 'a' || c > 'z') {
         return c;
       }
-      c = str[CharOrdLow(c)];
+      c = (char_u)str[CharOrdLow(c)];
```

Comment:
as this is the only usage of str, we could just as well declare it as `char_u` directly (the number of char_u doesn't change..)